src/app/shared/pipes/pom-common-pipes.module.ts

```diff
@@ -1,10 +1,13 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { PluralizePolishPipe } from './pluralize-polish/pluralize-polish.pipe';
+import { PathPipe } from './path/path.pipe';
+
+export const PIPES = [PluralizePolishPipe, PathPipe];
 
 @NgModule({
-  declarations: [PluralizePolishPipe],
-  exports: [PluralizePolishPipe],
+  declarations: PIPES,
+  exports: PIPES,
```

Comment:
I never know what actually works here given that Angular statically analyzes NgModule declarations during the build and dynamic code often doesn't work properly...

Reply:
done